<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindCustomerWindowClass</class>
 <widget class="QMainWindow" name="FindCustomerWindowClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>73</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find Customer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="IDLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IDTextBox">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>10</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
